dev/devicelab/lib/framework/ios.dart

```diff
@@ -102,6 +104,40 @@ Future<bool> containsBitcode(String pathToBinary) async {
   return !emptyBitcodeMarkerFound;
 }
 
+Future<bool> bonjourServiceFound(String appBundlePath) async =>
+  (await eval(
+    'plutil',
+    <String>[
+      '-extract',
+      'NSBonjourServices',
+      'xml1',
+      '-o',
+      '-',
+      path.join(
+        appBundlePath,
+        'Info.plist',
+      ),
+    ],
+    canFail: true,
+  )).contains('_dartobservatory._tcp');
+
+Future<bool> localNetworkUsageFound(String appBundlePath) async =>
+  await exec(
+    'plutil',
```

Comment:
oh lol, TIL. So there's an actual built-in tool to turn plists to things like json so it can be more easily consumed by things like dart?

Reply:
Yup, it's being used for other plist parsing, see https://github.com/flutter/flutter/blob/80619f100ffefec82528be1b92a76ad22807e459/packages/flutter_tools/lib/src/ios/plist_parser.dart#L53.  Except the json part doesn't totally work... https://github.com/flutter/flutter/issues/62160

Reply:
oh lol, sad